docs/tutorials/flyscanning/json_writer.py

```diff
@@ -0,0 +1,29 @@
+class JSONWriter:
```

Comment:
Thans, Dan! I think jsonlines is actually what was there initially (I am using the piece of code you have written during one of my first days here when you were explaining Bluesky to me). As far as I remember, I had some difficulties with opening jsonl with VS code or maybe it didn't format it correctly (probably I just needed to install another extension), but it worked with json. I like the selling points though!

On this note, do you think it would make sense to add these callbacks to Bluesky directly?

Reply:
Definitely! A `bluesky.callbacks.jsonl` module would be great to have.

We actually ship this functionality as a "suitcase", [`suitcase-jsonl`](https://github.com/bluesky/suitcase-jsonl) but the amount of framework here is overkill for this use case. Better to ship a simple `JSONLWriter` in bluesky. Perhaps in the docs we can recommend a suitable VSCode plugin?